### ISSUE TITLE
htmlnano configuration

### ISSUE THREAD

**Issue by hsablonniere** [hsablonniere/hsablonniere.com#13]
I tried many different configurations. I know I have a very simple context but sometimes the result on gzip and brotli vs results on raw is surprizing.

```
// Nothing activated

 22614 index.html
  8197 index.html.gz
  6842 index.html.br

// Not really effective in this context:
collapseAttributeWhitespace: true,
collapseBooleanAttributes: true,
deduplicateAttributeValues: true,
removeComments: true,
removeEmptyAttributes: true,
removeRedundantAttributes: true,

 22614 index.html
  8197 index.html.gz
  6842 index.html.br

// vv Doesn't change much in this context
// With 'alphabetical' => +5 bytes on brotli, -3 bytes on gzip
sortAttributesWithLists: 'alphabetical',

 22614 index.html
  8194 index.html.gz
  6847 index.html.br

// go back with sortAttributesWithLists: false,
removeAttributeQuotes: true,

 22280 index.html
  8167 index.html.gz
  6865 index.html.br

// go back with removeAttributeQuotes: false,
collapseWhitespace: 'conservative',

 22503 index.html
  8136 index.html.gz
  6788 index.html.br

collapseWhitespace: 'all',

 22284 index.html
  8114 index.html.gz
  6779 index.html.br

// vv Not really useful
// With true => small impact on raw, tiny impact on gzip, bigger size on brotli
mergeStyles: true,

 22269 index.html
  8111 index.html.gz
  6780 index.html.br

// go back with mergeStyles: false,
Comparison before / after

 22614  22269  345  1.53%  index.html
  8197   8111   86  1.05%  index.html.gz
  6842   6780   62  0.91%  index.html.br
```

In the end, I would mostly use htmlnano for `collapseWhitespace: 'all'` and the gain is just a joke with a result that is not view-source friendly.

**Comment by hsablonniere** [comment]
If I wanted to be really view source friendly, I could use https://github.com/posthtml/posthtml-beautify instead as an "optimizer". The `Only double quotes` could break if I introduce Web components with JSON attributes.

**Comment by hsablonniere** [comment]
Fixed with #14
